project/Hooje/README.md

```diff
@@ -0,0 +1,59 @@
+Simple Parallel Decision Tree in C++
+====
+
+# Basic Information
+Traditional Machine Learning models are still effective ways for classification.
+And Decision Tree is one of them.
+
+Decision Tree is a series a node that start with one root node and extends to many nodes that can represents the classfication rules for the data.
+For a dataset, Decision Tree will calculate the impurity (GINI index, entropy...) of every possible split, and choose the best one to be the rule of root node and create two leaf nodes.
+For each node, use the same method to decide the rule and create its child nodes.
+
+
+# Problem to Solve
+Decision Tree is an effective model in many classification senerio. However, it takes so much time to run because it cannot be speedup by CUDA like Neural Network, especially when there are a lot of features in that Dataset.
+
+Thus, I want to implement a simple version of Decision Tree in C++ (can be used from Python) from scratch to reduce the execution time. Besides, I would like to modify some of its functions so that it can be executed parallelly.

```

Comment:
Do you plan to make it multi-thread?  Concurrent programming is usually more complex than the scope of this course, but you may try.

You may also consider to install SIMD in the code and measure the runtime difference.  This is also challenging.

Reply:
Thanks, I will compare the results between with and without SIMD.
After that, I will try my best to finish multi-thread version and compare it with them.

Reply:
That sounds like a plan.  Please add it in the proposal.